build.gradle

```diff
@@ -129,6 +130,7 @@ dependencies {
     compile 'com.github.wyukawa.elasticsearch.unofficial.jdbc.driver:elasticsearch-jdbc-driver:0.0.9'
     compile 'org.jsoup:jsoup:1.11.3'
     compile 'mysql:mysql-connector-java:5.1.47'
+    compile 'com.owlike:genson:1.6'
```

Comment:
I'm not familiar with JSON library but is genson necessary?

Reply:
This library is needed when we set json key name explicitly.
eg `@JsonProperty("formattedQuery")`